erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -624,7 +624,7 @@ def get_item_details(item, project = None):
 	return res
 
 @frappe.whitelist()
-def make_work_order(bom_no, item, qty=0, project=None):
+def make_work_order(bom_no, item, manufacturing_type, finished_good_qty=0, qty=0, project=None):
```

Comment:
Always add new arguments to the end of the function, to avoid breaking existing calls for this function